app/components/Wallet/ReceiveModal/ReceiveModal.js

```diff
@@ -11,25 +11,46 @@ import messages from './messages'
 
 const QRCODE_TYPE_ADDRESS = 'address'
 const QRCODE_TYPE_PUBKEY = 'pubkey'
+const QRCODE_TYPE_LNDCONNECT = 'lndconnect'
+
+class ReceiveModal extends React.PureComponent {
+  static defaultProps = {
+    activeWalletSettings: {}
+  }
 
-class ReceiveModal extends React.Component {
   state = {
     qrCodeType: QRCODE_TYPE_ADDRESS
   }
 
-  copyPubkeyToClipboard = () => {
-    const { pubkey, intl } = this.props
-    copy(pubkey)
-    const notifTitle = intl.formatMessage({ ...messages.pubkey_copied_notification_title })
-    const notifBody = intl.formatMessage({ ...messages.pubkey_copied_notification_description })
-    showNotification(notifTitle, notifBody)
-  }
+  copyToClipboard = () => {
+    const {
+      pubkey,
+      address,
+      activeWalletSettings: { lndConnect },
+      intl
+    } = this.props
+    const { qrCodeType } = this.state
+
+    let qrCode, notifTitle, notifBody
+    switch (qrCodeType) {
```

Comment:
I would add `default` case also

Reply:
The default case for these is that none of them are set, which prevents this QR code from displaying. If one of the predefined QR codes is set then all 3 of these values will be set as per this switch statement. So think the default case is implicit here.